.babelrc

```diff
@@ -0,0 +1,15 @@
+{
+    "presets": [
+        "@babel/react",
+        "@babel/preset-flow",
+        [
+            "@babel/env",
+            {
+                "targets": {
+                    "browsers": "last 2 versions"
+                }
+            }
+        ]
+    ],
+    "plugins": ["@babel/plugin-proposal-class-properties"]
```

Comment:
The Docker build is broken, because packages are installed to ``/tools/node_modules``. I was able to fix this in ``webpack.config.js``, because code was available, but I'm not sure how to fix this in a JSON config.

Reply:
Hmmmm. Let me look at this.

Reply:
We can swith to using `babel.config.js` and then probably use the same solution as in Webpack config. Will have a look.